pkg/ingress/apisix_tls.go

```diff
@@ -146,7 +146,7 @@ func (c *apisixTlsController) sync(ctx context.Context, ev *types.Event) error {
 		c.controller.recordStatus(tls, _resourceSyncAborted, err, metav1.ConditionFalse)
 		return err
 	}
-
+	c.controller.metricsCollector.IncrSyncSuccess("ssl")
```

Comment:
I saw the same logic was put inside `handleSyncErr` while here you put it in another place, we'd better keep them consistent.

Reply:
now similar logic was all put inside `handleSyncErr`  to keep consistent.